python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -565,7 +565,7 @@ def get_entries(
             )
 
     def get_entries_iter(
-        self, *, start_timestamp: int = 0, end_timestamp: int = 0
+        self, *, start_timestamp: int = 0, end_timestamp: int = 0, next_: str = ""
```

Comment:
blocking: next_ is a bit cryptic, perhaps we can do `checkpointed_paging_url` and then do

```
next_ = checkpointed_paging_url
```